README.md

```diff
@@ -1,13 +1,11 @@
 # AWS managed OpenTelemetry Lambda Layers
 
 As a downstream Repo of [opentelemetry-lambda](https://github.com/open-telemetry/opentelemetry-lambda), ___aws-otel-lambda___ publishes AWS managed OpenTelemetry Lambda layers that are preconfigured for use with AWS services and bundle the reduced AWS Collector. Users can onboard to OpenTelemetry in their existing Lambda functions by adding these ready-made layers directly.
-- Python3.8 layer [aws-otel-python38-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains OpenTelemetry Python `v1.5.0` with Contrib `v0.24b0`
-- Nodejs layer [aws-otel-nodejs-ver-0-24-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript `v0.24.0` with Contrib `v0.24.0`
-- Java-Wrapper layer [aws-otel-java-wrapper-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) contains OpenTelemetry Java `v1.5.0`
-- Java-Agent layer [aws-otel-java-agent-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java-auto-instr) contains AWS OpenTelemetry Java instrumentation `v1.5.0`
-- Collector layer [aws-otel-collector-ver-0-33-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-dotnet) contains ADOT Collector for Lambda `v0.12.0`
-
-
+- Python3.8 layer [aws-otel-python38-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains AWS Python Extension `v1.0.1` which uses OpenTelemetry Python `v1.5.0`
```

Comment:
Just a suggestion to alternative wording, I think the `OpenTelemetry`... with the `AWS Python Extension` helps imply that "AWS Python Extension" has to do with OpenTelemetry? Otherwise I would think it's something separate. But the current wording works too.

```suggestion
- Python3.8 layer [aws-otel-python38-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains OpenTelemetry Python `v1.5.0` with the AWS Python Extension `v1.0.1`
```

Reply:
I actually originally worded it that way but changed it bc my understanding was OpenTelemetry Python was a dependency of the AWS OpenTelemetry Python Extension so wasn't sure if my original wording made sense. But if you think it makes sense this way then I'm happy to change it!

Reply:
> my understanding was OpenTelemetry Python was a dependency of the AWS OpenTelemetry Python Extension

Yes this understanding is correct 🙂 

I _do_ think this wording is a worthwhile change! Let me know if it can be worded differently than what I said above 😄

Reply:
> I do think this wording is a worthwhile change!

Sorry, do you think the current change or your proposed change is worthwhile? haha